docs/security-considerations.rst

```diff
@@ -598,9 +599,10 @@ types.
 
 ::
 
+    // This will report a warning if no SMT solver is available.
```

Comment:
Might need to be after the pragma.

Reply:
(see ``scripts/isolate_tests.py`` L55+ - the extracted file will only start at the "pragma solidity")

Reply:
Why? It's weird that it fails here, it passes locally

Reply:
Ah, your comment took a while to show up...
Updated